api-reference/50 Common/utils/viz/registerGradient(type_options).md

```diff
@@ -5,22 +5,185 @@ export: registerGradient
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Registers a new gradient.
 
 ##### return: String
-<!-- Description goes here -->
+The fill id.
```

Comment:
Since you use these very short descriptions, I would go without any article. 

You can say "Specifies the fill ID for this gradient." But when you only say "Fill ID", then you can go without an article. The article in the sentence tell us that it's "that exact fill id that belongs to THIS gradient". But when you only say one or two words that describe the kind of value - "width", "color", "fill id", I don't think you need articles. 

```suggestion
Fill ID.
```